build_tools/compiler/mustache_inheritance_processor.rb

```diff
@@ -1,6 +1,10 @@
 require 'erb'
 require_relative 'template_processor'
 
+$fontsCSS = '<!--[if IE 8]><link href="{{{ assetPath }}}stylesheets/fonts-ie8.css?0.22.3" media="all" rel="stylesheet" /><![endif]-->
+    <!--[if gte IE 9]><!--><link href="{{{ assetPath }}}stylesheets/fonts.css?0.22.3" media="all" rel="stylesheet" /><!--<![endif]-->
+    <!--[if lt IE 9]><script src="{{{ assetPath }}}javascripts/ie.js?0.22.3"></script><![endif]-->'
+
```

Comment:
This is probably not the right place to store this, anybody have any suggestions?

Reply:
I don't think that ie.js is related to the fonts.. 🤔

Reply:
(Also you could probably get away with having it inline below...)

Reply:
Whoops about the IE stuff